packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -324,6 +324,38 @@ export function getPublishResolver<TSource, TContext, TArgs>({
   }
 }
 
+/**
+ *  Returns values for link arguments, also covers the cases for
+ * if the link parameter contains constants that are appended to the link parameter
+ *
+ * e.g. instead of:
+ * $response.body#/employerId
+ *
+ * it could be:
+ * abc_{$response.body#/employerId}
+ */
+function inferLinkArguments (paramName, value, resolveData, source, args) {
+  if (Object.prototype.toString.call(value) === '[object Object]') {
+    return Object.entries(value).reduce((acc, [key, value]) => {
+      acc[key] = inferLinkArguments(paramName, value, resolveData, source, args)
+      return acc
+    }, {})
+  }
+
+  if (value.search(/{|}/) === -1) {
```

Comment:
... namely, we cannot assume that `value` is a string. There seems to be a mismatch with our type definitions and OAS.